bins/revm-test/src/bin/burntpix/static_data.rs

```diff
@@ -0,0 +1,22 @@
8816114ce88518a5180029080020161187d565b931b828401610ca1565b03928203020501906114f06113bf8451865190611914565b8161150382800264e8d4a510000361187d565b84020592020583528152610e07565b915080916114b7565b969796601381036115b3575050505050505061157d90622fefd8945164e8d4a51000958187920202059081156115ab575b81811d92839161155f89518b5190611914565b901d9261157489518b5180029080020161187d565b95518401610ca1565b131561159d57611591916111e791036118ae565b02059083528152610e07565b611591916111e791016118ae565b84915061154c565b601491939597999a50809294969850145f1461161357505050505050509080610e5a916115e88551875180029080020161187d565b0190811561160b575b61160082828751871b02610c97565b938651901b02610c97565b9050806115f1565b60158103611665575050505050505050610e5a825182623d0900818751800284800201050191821561165b575b61164f91839160021b02610c97565b92855160021b02610c97565b9091508190611640565b601681036116875750505050505050505061168081516118ae565b8152610e07565b601781036116f05750505050505050506116dc6116a484516118ae565b826116b782800264e8d4a510000361187d565b9182156116e6575b6116cb91839102610c97565b926116d685516118ae565b02610c97565b9083528152610e07565b90915081906116bf565b6018819c9495969799989c145f14611729575050505090878181869486020880940208961d92838388831c070386521c07038152610e07565b9350945094955097506019915014611744575b505050610e07565b82918183920290800203058002058015611786575b801561096e5761176f9064e8d4a510000561187d565b8251845182028390058552020581525f808061173c565b5080611759565b80515f5b81811061179d57505050565b60018082015b8381106117b4575050600101611791565b81906117c08487610a82565b516117cb8288610a82565b51126117d8575b016117a3565b6117e28187610a82565b516117ed8588610a82565b516117f88389610a82565b526118038588610a82565b526117d2565b5f905b6216e36081101561186a57600190620f423f190181815b600a8410611832575050505090565b90918060028596611844839786610964565b0193620f424093849102049361185c86890186610964565b900396019493920204611823565b9062062fd9600391019160011b0461180c565b90600180830160011d90835b84831261189557505050565b91935090836118a48183610c97565b01821d9190611889565b905f91625fdfb0809107905f821261190d575b50600260018181845b8386136118d957505050505050565b9091929364e8d4a510008280889a6118f489869b8902610c97565b019a0202059594600185018502029301915f03906118ca565b015f6118c1565b91908215806119c7575b6119c1575f9061192d84611a34565b61193682611a34565b80821384146119a95761199191620f42406119519202610c97565b620f42409081808280020581808080612cc81985020561caee018402056201c41019018302056202f2d3018202056205131719010205620f422601020590565b622fefd860011d03935b02126119a357565b905f0390565b61195190620f42406119bb9302610c97565b9361199b565b505f9150565b50801561191e565b6119db6119ff916118ae565b6119ed81800264e8d4a510000361187d565b908115611a02575b620f424002610c97565b90565b620f424091506119f5565b6001600160401b91675851f42d4c957f2d020890621e8480602083901c07620f423f190190565b5f81136119ff575f039056fec65a7bb8d6351c1cf70c95a316cc6a92839c986682d98bc35f958f4883f9d2a8a2646970667358221220b3f2fe21ac0faf4ab1951c6297ccc37675d0c82e067ed2dd736ca86eeb03ebe164736f6c63430008170033");
```

Comment:
It's best to follow clippy suggestions and use `static` instead.

Reply:
here: https://github.com/bluealloy/revm/pull/1016